content/posts/batching-requests.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "Batching Requests in APISIX Ingress"
```

Comment:
Nitpick: why quotes?

Reply:
I am used to using it! I try to not use it on APISIX/API7 sites though 😅

Reply:
If it's not useful, I'd advise you to avoid it. Engineering mindset 😉